code/modules/awaymissions/maploader/reader.dm

```diff
@@ -222,7 +271,11 @@ var/global/dmm_suite/preloader/_preloader = new
 
 	var/turf/T = locate(x,y,z)
 	if(T)
-		instance = new path (T)//first preloader pass
+		if(ispath(path, /turf))
+			T.ChangeTurf(path, 1)
+			instance = T
+		else
+			instance = new path (T)//first preloader pass
 
```

Comment:
The system works without ruining all the pipes if I leave out the `ChangeTurf` here, and revert it to what it was before, however: This is likely just a hacky solution. I know the pipes pop off if the nodes don't get properly created, but I note that `defer` is set - when investigating, and I find that all the `air` datums of the turfs in the away missions are completely empty. I'll investigate further.